modules/installation-configuration-parameters.adoc

```diff
@@ -499,16 +499,9 @@ ifdef::azure[]
 |====
 |Parameter|Description|Values
 
-|`machines.platform.azure.type`
-|The Azure VM instance type.
-|VMs that use Windows or Linux as the operating system. See the
-link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
-in the Azure documentation.
-
 |`machines.platform.azure.osDisk.diskSizeGB`
 |The Azure disk size for the VM.
-|Integer that represents the size of the disk in GB, for example `512`. The
-minimum supported disk size is `120`.
+|Integer that represents the size of the disk in GB. The minimum supported disk size is 1TB.
```

Comment:
Just a quick question whether 1TB should be written in terms of GB, if that's what needs to go into this field?

Reply:
thanks @bergerhoffer  i just saw in [this PR](https://github.com/openshift/installer/pull/2367)  that it needs to be `1024`. i'll fix it! thanks!